docs/source/guide/explanation/algorithms/classification/multi_label_classification.rst

```diff
@@ -52,11 +52,47 @@ In the table below the `mAP <https://en.wikipedia.org/w/index.php?title=Informat
+
+.. _mlc_cls_semi_supervised_pipeline:
+
+- ``BarlowTwins loss``: A specific implementation of Semi-SL that combines the use of a consistency loss with strong data augmentations, and a specific optimizer called Sharpness-Aware Minimization (SAM) to improve the performance of the model.
+
+- ``Adaptive loss auxiliary loss weighting``: A technique for assigning such a weight for an auxiliary loss that the resulting value is a predefined fraction of the EMA-smoothed main loss value. This method allows aligning contribution of the losses during different training phases.
+
+- ``Exponential Moving Average (EMA)``: A technique for maintaining a moving average of the model's parameters, which can improve the generalization performance of the model.
+
+- ``Additional techniques``: Other than that, we use several solutions that apply to supervised learning (No bias Decay, Augmentations, Early-Stopping, etc.)
+
+Please, refer to the :doc:`tutorial <../../../tutorials/advanced/semi_sl>` on how to train semi-supervised learning.
+Training time depends on the number of images and can be up to several times longer than conventional supervised learning.
+
+In the table below the mAP on some academic datasets using our pipeline is presented.
+
++-----------------------+---------+----------------------+----------------+---------+----------------+---------+
+|        Dataset        | AerialMaritime 3 cls |         | VOC 2007 3 cls |         | COCO 14 3 cls  |         |
++=======================+======================+=========+================+=========+================+=========+
+|                       |   SL                 | Semi-SL |  SL            | Semi-SL |   SL           | Semi-SL |
++-----------------------+----------------------+---------+----------------+---------+----------------+---------+
+| MobileNet-V3-large-1x |  74.28               |  74.41  | 96.34          |  97.29  |  82.39         |  83.77  |
++-----------------------+----------------------+---------+----------------+---------+----------------+---------+
+|   EfficientNet-B0     |  79.59               |  80.91  | 97.75          |  98.59  | 83.24          |  84.19  |
++-----------------------+----------------------+---------+----------------+---------+----------------+---------+
+|  EfficientNet-V2-S    |  75.91               |  81.91  | 95.65          |  96.43  | 85.19          |  84.24  |
++-----------------------+----------------------+---------+----------------+---------+----------------+---------+
+
+AerialMaritime was sampled with 5 images per label. VOC and COCO were sampled with 40 images per label.
+
+.. note::
+    This result can vary greatly depending on the image selected for each class. Also, since there are few labeled settings for the Semi-SL algorithm. Some models may require larger datasets for better results.
```

Comment:
```suggestion
    This result greatly depends on the image selected for each class. Also, since there are few labeled settings for the Semi-SL algorithm. Some models may require larger datasets for better results.
```